src/routes/safe/components/TransactionsNew/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const Transactions = (): React.ReactElement => <div>new txs</div>
```

Comment:
can we maybe decide between using `JSX.Element` or `React.ReactElement` as a type?

Reply:
do you have a suggestion?

Reply:
they’re the same if I understand correctly, so I don’t have strong preference. Maybe search across the project and see which one we use more 🙂

Reply:
We were using almost the same amount for the two cases, I just moved all the `JSX.Element` to `ReactElement `